resource/ui/MainWindow.ui

```diff
@@ -103,7 +103,7 @@
               <object class="GtkLabel" id="close_to_tray_label">
                 <property name="visible">True</property>
                 <property name="can-focus">False</property>
-                <property name="label" translatable="yes">Close To Tray</property>
+                <property name="label" translatable="yes">          Close To Tray          </property>
```

Comment:
Why? Please don't use spaces at the beginning and end, if needed use the appropriate policies.